fileattrs/python.attr

```diff
@@ -1,4 +1,4 @@
 %__python_provides	%{_rpmconfigdir}/pythondeps.sh --provides
 %__python_requires	%{_rpmconfigdir}/pythondeps.sh --requires
-%__python_path	^((/usr/lib(64)?/python[[:digit:]]\\.[[:digit:]]/.*\\.(py[oc]?|so))|(%{_bindir}/python[[:digit:]]\\.[[:digit:]]))$
+%__python_path	^((%{_libdir}(64)?/python[[:digit:]]\\.[[:digit:]]/.*\\.(py[oc]?|so|pyd))|(%{_bindir}/python[[:digit:]]\\.[[:digit:]]))$
```

Comment:
on x86_64 that would be: `/usr/lib64(64)?` which is wrong.. do you really have very different libdir or is it %{_prefix}/lib* ?